src/apis/OnTheDex.tsx

```diff
@@ -0,0 +1,25 @@
+const url = 'https://api.onthedex.live/public/v1'
```

Comment:
This code doesn't need to be included in this PR - it's not used anywhere

Reply:
I think we should add the TVL logic back into this PR @ckniffen 

product wants it to exist

Reply:
It won't show anything for the AMM Devnet - those tokens are all fake

Reply:
yea, but it has to be merged into devnet in order for it to be merged into mainnet right? We can have it as a separate PR.

Reply:
I've posted the onthedex changes in this branch: https://github.com/Amindv1/explorer-1/tree/AMMadditions

Reply:
Yes, but AMMs aren't going to be on mainnet anytime soon, so it's not urgent. Plus, there needs to be checks for the onthedex data to make sure it only populates on mainnet.

Reply:
removed it from this PR as well